packages/contracts-bedrock/test/DeployImplementations.t.sol

```diff
@@ -267,13 +266,142 @@ contract DeployImplementations_Test is Test {
+        vm.etch(address(srSystemConfigImpl), hex"01");
+        assertEq(srSystemConfigImpl, address(dio.systemConfigImpl()));
+
+        address srL1CrossDomainMessengerImpl = address(0xD3494713A5cfaD3F5359379DfA074E2Ac8C6Fd65);
+        vm.etch(address(srL1CrossDomainMessengerImpl), hex"01");
+        deployImplementations.deployL1CrossDomainMessengerImpl(dii, dio);
+        assertEq(srL1CrossDomainMessengerImpl, address(dio.l1CrossDomainMessengerImpl()));
+
+        address srL1ERC721BridgeImpl = address(0xAE2AF01232a6c4a4d3012C5eC5b1b35059caF10d);
+        vm.etch(address(srL1ERC721BridgeImpl), hex"01");
+        deployImplementations.deployL1ERC721BridgeImpl(dii, dio);
+        assertEq(srL1ERC721BridgeImpl, address(dio.l1ERC721BridgeImpl()));
+
+        address srL1StandardBridgeImpl = address(0x64B5a5Ed26DCb17370Ff4d33a8D503f0fbD06CfF);
+        vm.etch(address(srL1StandardBridgeImpl), hex"01");
+        deployImplementations.deployL1StandardBridgeImpl(dii, dio);
+        assertEq(srL1StandardBridgeImpl, address(dio.l1StandardBridgeImpl()));
+
+        address srOptimismMintableERC20FactoryImpl = address(0xE01efbeb1089D1d1dB9c6c8b135C934C0734c846);
+        vm.etch(address(srOptimismMintableERC20FactoryImpl), hex"01");
+        deployImplementations.deployOptimismMintableERC20FactoryImpl(dii, dio);
+        assertEq(srOptimismMintableERC20FactoryImpl, address(dio.optimismMintableERC20FactoryImpl()));
+
+        address srOptimismPortalImpl = address(0xe2F826324b2faf99E513D16D266c3F80aE87832B);
+        vm.etch(address(srOptimismPortalImpl), hex"01");
+        deployImplementations.deployOptimismPortalImpl(dii, dio);
+        assertEq(srOptimismPortalImpl, address(dio.optimismPortalImpl()));
+
+        address srDelayedWETHImpl = address(0x71e966Ae981d1ce531a7b6d23DC0f27B38409087);
+        vm.etch(address(srDelayedWETHImpl), hex"01");
+        deployImplementations.deployDelayedWETHImpl(dii, dio);
+        assertEq(srDelayedWETHImpl, address(dio.delayedWETHImpl()));
+
+        address srPreimageOracleSingleton = address(0x9c065e11870B891D214Bc2Da7EF1f9DDFA1BE277);
+        vm.etch(address(srPreimageOracleSingleton), hex"01");
+        deployImplementations.deployPreimageOracleSingleton(dii, dio);
+        assertEq(srPreimageOracleSingleton, address(dio.preimageOracleSingleton()));
+
+        address srMipsSingleton = address(0x16e83cE5Ce29BF90AD9Da06D2fE6a15d5f344ce4);
+        vm.etch(address(srMipsSingleton), hex"01");
+        deployImplementations.deployMipsSingleton(dii, dio);
+        assertEq(srMipsSingleton, address(dio.mipsSingleton()));
+
+        address srDisputeGameFactoryImpl = address(0xc641A33cab81C559F2bd4b21EA34C290E2440C2B);
+        vm.etch(address(srDisputeGameFactoryImpl), hex"01");
+        deployImplementations.deployDisputeGameFactoryImpl(dii, dio);
+        assertEq(srDisputeGameFactoryImpl, address(dio.disputeGameFactoryImpl()));
+    }
+
+    function test_deployAtNonExistentRelease_reverts() public {
```

Comment:
this test proves that you won't accidentally deploy implementations if the `op-contracts` tag doesn't exist.